aait/backend/group-2B/Persistence/Repositories/GenericRepository.cs

```diff
@@ -6,7 +6,7 @@ namespace SocialSync.Persistence.Repositories;
 public class GenericRepository<T> : IGenericRepository<T>
     where T : class
 {
-    private SocialSyncDbContext _dbContext;
+    protected SocialSyncDbContext _dbContext;
```

Comment:
The dbcontext should be private readonly

Reply:
How will the inheriting classes access it?

Reply:
You can inject the dbcontext for all the repositories that inherit from the GenericRepository.